addons/mail/static/src/components/notification_list/notification_list.js

```diff
@@ -112,7 +112,7 @@ class NotificationList extends Component {
                     return 1;
                 }
                 if (t1.lastMessage && t2.lastMessage) {
-                    return t1.lastMessage.date.isBefore(t2.lastMessage.date) ? 1 : -1;
+                    return t1.lastMessage.id < t2.lastMessage.id ? 1 : -1;
```

Comment:
date is still used in `threadNeedactionNotifications` in this file (can't put the comment on the correct line because there is no diff on it).